### PR TITLE
fix & added personality check

### PR DESCRIPTION
**Pull request by AnyoneLL**

### CODE REVIEW

File: common/scripted_effects/LIDA_dom_effects.txt [hunk 1]
```diff
@@ -479,6 +479,126 @@ calculate_dom_success_effect = {
 			}
 		}
 	}
+    save_scope_value_as = {
```
Comment (PerplexedPeach) [inline common/scripted_effects/LIDA_dom_effects.txt:482]: Can you fix the formatting here? The spacing is off

Comment (AnyoneLL) [reply common/scripted_effects/LIDA_dom_effects.txt:482]: That's weird. It's perfectly fine on my IDE, it only appears like that on GitHub.

image

Comment (AnyoneLL) [reply common/scripted_effects/LIDA_dom_effects.txt:482]: Oh, don't tell me it's supposed to be whitespaces instead of tabs?

Comment (AnyoneLL) [reply common/scripted_effects/LIDA_dom_effects.txt:482]: Fixed

**Pull request merged**